docs/source/subprojects.rst

```diff
@@ -0,0 +1,136 @@
+.. _subprojects:
+
+
+===================
+Jupyter subprojects
+===================
+
+Project Jupyter is developed as a set of subprojects. The following is a topical list of the
+officially supported and maintained subprojects with links to the documentation or GitHub repo of
+each.
+
+
+Jupyter applications
```

Comment:
I'm not a fan of "Application", nbconvert is for me an application, but you seem to focus on GUI.